templates/registration/logged_out.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <form action="{% url 'logout' %}" method="post">
+    {% csrf_token %}
```

Comment:
This part of the code is redundant, because when the user gets to this page, they have already clicked on the logout button in the sidebar.